src/scenes/Organizations/Create/CreateOrganization.tsx

```diff
@@ -10,23 +12,42 @@ import {
 export const CreateOrganization = (props: Except<Props, 'onSubmit'>) => {
   const [createOrg] = useCreateOrganizationMutation();
   const { enqueueSnackbar } = useSnackbar();
-
+  const [close, setClose] = useState<boolean>(false);
```

Comment:
The generic can be inferred here.
```suggestion
  const [close, setClose] = useState(false);
```